### ISSUE TITLE
Connection broken for few seconds after locking/unlocking the screen

### ISSUE THREAD

**Issue by Vizzard** [synergy/synergy#4485]
Synergy 1.6.3 x64 on both Windows (server) and Ubuntu 14.10 (client).

Can't move cursor to client after locking server screen. After few seconds it reconnects and I'm able to move cursor between client and server.
Same after unlocking Windows screen. For few seconds after unlocking I'm not able to move cursor to client. Then it reconnects and everything works.

Server log:
activeDesktop:Default
WARNING: detected application not running, pid=285628
activeDesktop:Winlogon
ERROR: failed to get desktop path, no drop target available, error=2
WARNING: detected application not running, pid=293356
activeDesktop:Default

Client log:
NOTE: disconnected from server
NOTE: connecting to '192.168.0.7': 192.168.0.7:24800
NOTE: connected to server

If it helps more I can put some debug level log but I guess it's easy to reproduce.

**Comment by the-wes** [comment]
This is how it is currently designed to work. The other option is to not be able to switch screens at all, so this is a step forward. There are probably ways to continue to improve this, but this would be a feature request.

**Comment by nbolton** [comment]
Yeah, its by design, but the design is pretty cheap (my bad, sorry). Causes a lot of downtime and isn't very seamless.
